tests/vm-migration

```diff
@@ -53,9 +53,16 @@ for _ in $(seq 60); do
   fi
 done
 
-# Launch two instances for our LXD cluster and wait for them to be ready.
-lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member1 --vm -c limits.memory=2GiB
-lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member2 --vm -c limits.memory=2GiB
+# Launch two instances for our LXD cluster and wait for them to be ready. If the host supports `devlxd_images_vm` then
+# set `security.devlxd.images=true` so that we don't have to download the image again.
+if hasNeededAPIExtension devlxd_images_vm; then
+  lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member1 --vm -c security.devlxd.images=true
+  lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member2 --vm -c security.devlxd.images=true
+else
+  lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member1 --vm
+  lxc launch "${TEST_IMG:-ubuntu-minimal-daily:24.04}" member2 --vm
```

Comment:
The GHA runner VM have 16G of RAM (https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) so you can probably allocate more RAM to those memberX VMs.

Have you already tried with `-c limits.cpu=2 -c limits.memory=4GiB`?

Reply:
I've bumped all the resources to what is used in microcloud and it's still not booting after over an hour. I'll go back to trying with containers and see how I get on.

Reply:
yes this would avoid needing to keep setting up the websockets over vsock for the exec sessions